charts/grafana/templates/service.yaml

```
@@ -48,6 +48,9 @@ spec:
   {{- with .Values.service.externalTrafficPolicy }}
   externalTrafficPolicy: {{ . }}
   {{- end }}
+  {{- if .Values.service.seessionAffinity }}
+  sessionAffinity: {{ .Values.service.sessionAffinity }}
```

For consistency reasons, it should be

```suggestion
  {{- with .Values.service.seessionAffinity }}
  sessionAffinity: {{ . }}
```

like the line above.